cfme/base/ui.py

```diff
@@ -620,19 +620,17 @@ def is_displayed(self):
         return (
             self.in_configuration and
             self.summary.is_displayed and
-            self.title.text == 'VMDB Summary'
```

Comment:
Question:
we are not getting any common `title` for all `tabs`?

Reply:
Yes. For the `ServerDatabaseView` we need to be sure we are in the Database section, for that I added `self.accordions.database.is_opened` below.
The title is different for each tab in this section and it's checked in the respective view for the tab, L627-665.